src/bindings/python/src/openvino/frontend/tensorflow/graph_iterator.py

```diff
@@ -0,0 +1,70 @@
+# Copyright (C) 2018-2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+# flake8: noqa
+# mypy: ignore-errors
+
+import tensorflow as tf
+from openvino.frontend.tensorflow.py_tensorflow_frontend import _FrontEndPyGraphIterator as GraphIterator
+
+from .node_decoder import TFGraphNodeDecoder
+
+
+class GraphIteratorTFGraph(GraphIterator):
```

Comment:
please leave a comment about `tf.Graph`. Does it have ordered inputs and outputs, etc.?

Reply:
Added comment.